sim/warrior/talents.go

```diff
@@ -15,8 +15,8 @@ func (warrior *Warrior) ToughnessArmorMultiplier() float64 {
 func (warrior *Warrior) ApplyTalents() {
 	warrior.AddStat(stats.MeleeCrit, core.CritRatingPerCritChance*1*float64(warrior.Talents.Cruelty))
 	warrior.ApplyEquipScaling(stats.Armor, warrior.ToughnessArmorMultiplier())
-	warrior.PseudoStats.BaseDodge += 0.01 * float64(warrior.Talents.Anticipation)
-	warrior.PseudoStats.BaseParry += 0.01 * float64(warrior.Talents.Deflection)
+	warrior.AddStat(stats.Dodge, 0.01*float64(warrior.Talents.Anticipation))
```

Comment:
Anticipation should be a defense stat.  Also as what Vigo said, defense stat should be calculated into the attack table.